gradle/libs.versions.toml

```diff
@@ -1,15 +1,15 @@
 [versions]
-agp = "8.2.2"
-kotlin = "2.1.0"
-multiplatform-settings = "1.2.0"
+agp = "8.6.0-alpha07" # Latest supported version
```

Comment:
Do you need an alpha version?

Reply:
Not really, but it's apparently the highest supported version. Anyway, I changed it to the previous non-alpha version, 8.5.2 👍